src/components/Chips/Chips.tsx

```diff
@@ -183,72 +190,81 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
           style: backgroundColorStyle
         }
       : {
+          tabIndex: isOverflowing ? 0 : undefined,
```

Comment:
I think the isOverflowing logic make the code more complicated and not organized (there is logic here for tooltip and for focus which depend on this).
I think it will be more organized if we will create a hook in the chips folder which called: useOverflowTooltip which return wrapperProps and tooltipProps.

Reply:
Did it, thanks!